assets/scss/templates/_signup.scss

```diff
@@ -83,28 +82,15 @@
           width: 100%;
         }
     }
-
-  }
-
-  .bg--signup {
-    background-image: url("../img/yellow-violet.png");
-    background-size: cover;
-
-    h1, h6, p {
-      color: $dp-color-brown;
+    .feature-panel--bg {
+      background-size: cover;
+      padding: $dp-spaces--lv5 $dp-spaces--lv10;
+      h1, h6, p {
+        color: $dp-color-brown;
+      }
     }
-
   }
 
-  .bg--login {
-    background-image: url("../img/violet-yellow.png");
-    background-size: cover;
-
-    h1, h6, p {
-      color: $dp-color-white;
-    }
-
-  }
 
   .bg--forgot {
     background-image: url("../img/violet-yellow.png");
```

Comment:
In this case (forgot password), the same thing as the start of the session will be repeated, so it might be better to adapt it.

Reply:
now i apply this change

Reply:
Excellent!! Nice job @fcoronelmakingsense !!